### DISCUSSION TITLE
MinIO Cluster start failed when all node start at same time

### DISCUSSION THREAD

**Discussion by creeew** [minio/minio#20597] [Q&A]
I have 4 node, and start them at the same time, from the log, every node print 
`Error: Read failed. Insufficient number of drives online (*errors.errorString)
Oct 31 09:30:22 minio2 minio[587]: 6: internal/logger/logger.go:258:logger.LogIf()
Oct 31 09:30:22 minio2 minio[587]: 5: cmd/prepare-storage.go:241:cmd.connectLoadInitFormats()
Oct 31 09:30:22 minio2 minio[587]: 4: cmd/prepare-storage.go:301:cmd.waitForFormatErasure()
Oct 31 09:30:22 minio2 minio[587]: 3: cmd/erasure-server-pool.go:108:cmd.newErasureServerPools()
Oct 31 09:30:22 minio2 minio[587]: 2: cmd/server-main.go:696:cmd.newObjectLayer()
Oct 31 09:30:22 minio2 minio[587]: 1: cmd/server-main.go:530:cmd.serverMain()
Oct 31 09:30:22 minio2 minio[587]: Waiting for a minimum of 2 drives to come online (elapsed 14m12s)`
Looks like they all in a init state and want other node could be readed.

And I restart every node in sequence, then the cluster could work.

In a MinIO cluster, is it not allowed to start all nodes simultaneously, and must they be started sequentially? Why?

**Comment by klauspost** [comment]
Nothing happens. This is normal.

**Reply by creeew** [reply]
you mean we have to make sure minio cluster node restart sequencially when the cluster restart?

**Reply by klauspost** [reply]
The cluster didn't fail. It works fine once it can connect to the other nodes.
